src/Raven.Client/Json/Serialization/NewtonsoftJson/Internal/Converters/DateOnlyConverter.cs

```diff
@@ -13,11 +13,13 @@ internal sealed class DateOnlyConverter : JsonConverter
         // More info available at: https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#Roundtrip
 
         public static readonly DateOnlyConverter Instance = new();
+
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
         {
-            if (value is null)
+            if (value == null)
```

Comment:
Is there a difference between them?

Reply:
None.

https://sharplab.io/#v2:C4LghgzgtgNAJiA1AHwAICYCMBYAUKgZgAIMiBhIgbzyNpOIHsAjAKwFMBjYIgDyIF4iAOwCuAGzEBuGnUIkALEQCyACgCUVWjLp0AlgDMiKvv0GiJa7TqpXrdW3QC+V57h1W5qRQDl1V6m52RAZGfLoQwuJiloF2lK46ro5AA==